### DISCUSSION TITLE
how to add self-defined label for target which not use static_configs

### DISCUSSION THREAD

**Discussion by like-inspur** [prometheus/prometheus#10098] [Q&A]
**What did you do?**
I want to monitor multi etcd cluster on kuberntes, and want to add self-define label as cluster flag for alert use
one etcd cluster running as static pod on mater node for kubernetes use, prometheus use node role kubernetes_sd_configs
another etcd cluster running as common pod on slave node for service use, prometheus use pod role kubernetes_sd_configs

**What did you expect to see?**
kubernetes_sd_configs can only use kubernetes default label for role node and pod, can'it add self-defined label like static_configs

**Comment by jeyrce** [comment]
@like-inspur Try releable_config  may help you

**Comment by like-inspur** [comment]
I have tried and found only use kubernetes defined label like node_name, namespace, can't use self defined label
